lib/nrf_modem_lib/lte_connectivity/lte_connectivity.h

```diff
@@ -36,10 +36,39 @@ enum lte_connectivity_if_down_options {
 	LTE_CONNECTIVITY_IF_DOWN_LTE_DISCONNECT
 };
 
+#define LTE_CONNECTIVITY_REG_TIMEOUT_NONE -1
+#define LTE_CONNECTIVITY_REG_TIMEOUT_INSTANT 0
+
 /** @brief Option types used to specify library behavior. */
 enum lte_connectivity_options {
 	/** Sets desired behavior when lte_connectivity_disable() / net_if_down() is called. */
 	LTE_CONNECTIVITY_IF_DOWN = 1,
+
+	/** Registration timeout.
+	 *
+	 *  After becoming unregisterred the serving cell, wait this long (in seconds) before
```

Comment:
```suggestion
	 *  After becoming unregisterred, the serving cell waits this long (in seconds) before
```

Reply:
These changes are from the first commit of this PR, which is cherry picked because they are not yet merged so I am resolving here

I'll add add you as a reviewer, and eventually make these changes even if you don't copy them over to that PR